CONTRIBUTING.md

```diff
@@ -151,7 +160,26 @@ features. For more information visit the [Babel documentation site]
 - https://saucelabs.com/beta/dashboard/tests
 - http://developer.samsung.com/remotetestlab/rtlDeviceList.action#
 
-#### Notes
+
+## Release management
+
+Ready to publish changes to npm?
+
+1. Ensure you're on `master` and `git pull` to confirm you're up-to-date.
+1. Export a personal access token called [`GITHUB_AUTH`](https://github.com/lerna/lerna-changelog#github-token).
+1. Run `yarn run changelog` and open `CHANGELOG.md` to see a preview
+of new changelog entries.
+We use a [tool](https://github.com/lerna/lerna-changelog#usage) that scans our
+Pull Requests for specific labels so if you see a PR missing from the changelog,
+ensure it has been labeled `breaking`, `enhancement`, `bug`,
+`documentation` or `internal`.
```

Comment:
Trivial: Would be nice to add three spaces of indentation to the additional lines in this long list item.

Reply:
https://github.com/cfpb/capital-framework/pull/910/commits/010a88fe075bbee0e6e5c425d28f49551dc90744